modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/OCamlClientCodegen.java

```diff
@@ -0,0 +1,703 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
@cgensoul please remove line 3 as per https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license/7301#7301